src/components/Recruitment/CreateRecruitmentInfoModal.tsx

```
@@ -1,17 +1,17 @@
-import { ChangeEvent, useState } from "react";
+import WarningIcon from "@/assets/warning.svg?react";
+import { QueryKey } from "@/constants/queryKey";
+import useCreateRecruitmentMutation from "@/hooks/mutations/useCreateRecruitmentMutation";
+import { RecruitmentModalInfoType } from "@/types/entities/recruitment";
+import { toKSTISOString } from "@/utils/validation/formatDate";
 import styled from "@emotion/styled";
```

ESLint

[error] `@emotion/styled` import should occur before import of `@/assets/warning.svg?react`

(import/order)